homeassistant/components/fyta/sensor.py

```diff
@@ -0,0 +1,171 @@
+"""Summary data from Fyta."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Final
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import PERCENTAGE, EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import FytaCoordinator, FytaEntity
+
+
+@dataclass(frozen=True)
+class FytaSensorEntityDescription(SensorEntityDescription):
+    """Describes Fyta sensor entity."""
+
+    value_fn: Callable[[str | int | float], str | int | float | datetime] = (
+        lambda value: value
+    )
+
+
+SENSORS: Final[list[FytaSensorEntityDescription]] = [
+    FytaSensorEntityDescription(
+        key="plant_name",
+        translation_key="fyta_plant_name",
+    ),
+    FytaSensorEntityDescription(
+        key="scientific_name",
+        translation_key="fyta_scientific_name",
+    ),
+    FytaSensorEntityDescription(
+        key="plant_status",
+        translation_key="fyta_plant_status",
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:flower",
+    ),
```

Comment:
What does this measure?

Reply:
FYTA provides a status on a scale 1 (too low) to 5 (too high) to give a high-level indication of the measurements.
https://fyta-io.notion.site/FYTA-Public-API-d2f4c30306f74504924c9a40402a3afd#22ac2c8137a64416865010a4e3b997f5

Reply:
Can we make these sensors return a string like "very_low" and make this an enum sensor with the options provided?